submissions/askuznetsov/html-css-popup/index.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML CSS Popup</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header__nav">
+        <ul class="header__nav__list">
+          <li class="header__nav__list__item">
+            <a href="" class="header__nav__list__item__link title">Mail</a>
+          </li>
+          <li class="header__nav__list__item">
+            <a href="" class="header__nav__list__item__link title">Images</a>
+          </li>
+          <li class="header__nav__list__item popup-svg">
+            <input
+              type="checkbox"
+              class="header__nav__list__item__checkbox"
+              id="popup__checkbox"
+              name="popup__checkbox"
+            />
+            <label for="popup__checkbox" class="header__nav__list__item__label">
+              <img
+                src="popup-icons/popup-button.png"
+                alt="popup-button"
+                class="header__nav__list__item__image popup__icon"
+              />
+            </label>
+            <div class="popup">
+              <div class="arrow"></div>
```

Comment:
You can also use pseudoelements for elements like this. It could simplify your layout

Reply:
Sorry, what do you mean by like this? popup class? or arrow? anyway, I don't understand what exactly pseudoelement you admit here.

Reply:
Instead of using additional elements in your layout, you can use pseudoelements

Reply:
Changed.